enterprise/backend/test/metabase_enterprise/sandbox/api/field_test.clj

```diff
@@ -30,24 +30,24 @@
                 {:gtaps      {:venues {:query      (mt.tu/restricted-column-query (mt/id))
                                        :remappings {:cat [:dimension (mt/id :venues :category_id)]}}}
                  :attributes {:cat 50}}]
-               ["native"
-                {:gtaps      {:venues {:query
-                                       (mt/native-query
-                                         {:query "SELECT id, name, category_id FROM venues WHERE category_id = {{cat}}"
-                                          :template-tags {"cat" {:id           "__MY_CAT__"
-                                                                 :name         "cat"
-                                                                 :display-name "Cat id"
-                                                                 :type         :number}}})
-                                       :remappings {:cat [:variable [:template-tag "cat"]]}}}
-                 :attributes {:cat 50}}]]]
+               #_["native"
```

Comment:
Did you mean to leave this commented out?

Reply:
Oops, fixed. Thanks